source/fundamentals/connection/connection-options.txt

```diff
@@ -208,8 +208,20 @@ parameters of the connection URI to specify the behavior of the client.
 
    * - **maxConnecting**
      - integer
-     - Specifies the number of connections the driver will driver concurrently
-       establish before making them available in the connection pool.
+     - Specifies the number of connections the driver concurrently
+       establishes before making them available in the connection pool.
```

Comment:
This piece of documentation was added in https://github.com/mongodb/docs-java/pull/194 and was not reviewed by the dev team. The current wording does not correctly describe what `maxConnecting` is. I propose to use the wording specified in the API documentation: https://mongodb.github.io/mongo-java-driver/4.4/apidocs/mongodb-driver-core/com/mongodb/connection/ConnectionPoolSettings.html#getMaxConnecting().